### PR TITLE
Feature/use original image ratio

### PR DESCRIPTION
**Pull request by kentis**

### CODE REVIEW

File: scripts/cp-editor.js [hunk 1]
```diff
@@ -2127,6 +2139,8 @@ H5PEditor.CoursePresentation.prototype.fitElement = function ($element, elementP
     style.top = elementParams.y + '%';
   }
 
+  console.log("applying style:");
+  console.log(style);
```
Comment (boyum) [inline scripts/cp-editor.js]: oopsie

Comment (kentis) [reply scripts/cp-editor.js]: ooops!

**Comment by boyum** [comment]
There are a few issues, still. First image is 400x225 (16:9), next is 225x400 (9:16), last is 400x400 
image

image
image
image

File: scripts/cp-editor.js [hunk 2]
```diff
@@ -1558,15 +1558,33 @@ H5PEditor.CoursePresentation.prototype.setImageSize = function (element, element
...
-  } else {
-    elementParams.height = (elementParams.height / element.$wrapper.innerHeight()) * 100;
-    elementParams.width = elementParams.height * fileRatio;
+  // if(elementParams.width > elementParams.height){
+  //   elementParams.width = (elementParams.width / element.$wrapper.innerWidth()) * 100;
+  //   elementParams.height = elementParams.width * (1/fileRatio)
+  // } else {
+  //   elementParams.height = (elementParams.height / element.$wrapper.innerHeight()) * 100;
+  //   elementParams.width = elementParams.height * fileRatio;
+  // }
```
Comment (boyum) [inline scripts/cp-editor.js:1571]: Can this be removed?

File: scripts/cp-editor.js [hunk 3]
```diff
@@ -1554,7 +1557,34 @@ H5PEditor.CoursePresentation.prototype.setImageSize = function (element, element
...
+
+  //Normalizse
+  // if(elementParams.width > elementParams.height){
+  //   elementParams.width = (elementParams.width / element.$wrapper.innerWidth()) * 100;
+  //   elementParams.height = elementParams.width * (1/fileRatio)
+  // } else {
+  //   elementParams.height = (elementParams.height / element.$wrapper.innerHeight()) * 100;
+  //   elementParams.width = elementParams.height * fileRatio;
+  // }
+
```
Comment (boyum) [inline scripts/cp-editor.js]: Can this be removed?

```suggestion
```

File: scripts/cp-editor.js [hunk 4]
```diff
@@ -1554,7 +1557,34 @@ H5PEditor.CoursePresentation.prototype.setImageSize = function (element, element
...
+  if(elementParams.height > element.$wrapper.innerHeight()){
+    elementParams.width = (element.$wrapper.innerHeight() * elementParams.width) / elementParams.height;
+    elementParams.height = element.$wrapper.innerHeight();
+  }
+  
+  elementParams.width = (elementParams.width / element.$wrapper.innerWidth()) * 100;
+  elementParams.height = (elementParams.height / element.$wrapper.innerHeight()) * 100;
+
+  console.log("post-normalization");
+  console.log(elementParams);
```
Comment (boyum) [inline scripts/cp-editor.js]:
```suggestion
```

**Review by boyum — APPROVED** [review]

**Pull request merged**